jep/227/compatibility.adoc

```diff
@@ -236,10 +236,10 @@ should be adjusted.
 |As of link:https://github.com/jenkinsci/plugin-util-api-plugin/releases/tag/plugin-util-api-1.2.5[1.2.5].
 
 |link:https://plugins.jenkins.io/purge-job-history/[purge-job-history]
-|Mostly compatible
+|Compatible
```

Comment:
No, mostly compatible _as of now_. (Compatible if and when that PR is released.)

Reply:
In this plugin, now you get a 500 if you are lacking permission and with the core update, you will get 403.
The PR corrects the bug that makes the "feature" working.

So if you consider such corner case in the "mostly", fine :) Just wanted to give details ;)